pkg/apis/resource/types.go

```diff
@@ -988,6 +988,14 @@ type ResourceClaimTemplateList struct {
 	Items []ResourceClaimTemplate
 }
 
+const (
+	AllocatedDeviceStatusMaxConditions        int = 8
+	AllocatedDeviceStatusDataMaxLength        int = OpaqueParametersMaxLength
```

Comment:
Let's use some actual value here, not `OpaqueParametersMaxLength`. If we ever change `OpaqueParametersMaxLength` it would be surprising if that also changed the limit for `AllocatedDeviceStatus.Data`, in particular when it's not documented as also applying there.

Reply:
Thank you, I added the actual value